docs/guides/edgemesh_test_env_guide.md

```diff
@@ -175,13 +162,15 @@ spec:
           privileged: true
 ```
 **note: -p: whitelist, only port in whitelist can go out from client to edgemesh then to server**
-- client request server: exec into client container and then run command: ```curl http://test-headless.default.svc.cluster:8080```, will get the response from server like: ```HOSTNAME:test-app-686c6dbf98-6hrdq IP:10.11.0.4```
+- client request server: exec into client container and then run command: ```curl http://edgemesh-example-service.default.svc.cluster:8080```, will get the response from server like: ```HOSTNAME:server-5c5868b79f-j4td7 IP:10.22.0.14```
```

Comment:
we can put the commands in `sh` snippets.

Reply:
put in ```bash``` snippets